pom.xml

```diff
@@ -209,6 +204,14 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
+            <groupId>jakarta.mail</groupId>
+            <artifactId>jakarta.mail-api</artifactId>
+            <version>2.1.2</version>
+            <type>jar</type>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Unrelated to the Prototype changes, but in Jenkins plugins we typically prefer plugin-to-plugin dependencies for libraries that are packaged as Jenkins plugins:

```xml
<dependency>
    <groupId>io.jenkins.plugins</groupId>
    <artifactId>jakarta-mail-api</artifactId>
    <version>2.0.1-3</version>
</dependency>
```

The reasoning is explained [here](https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/) and can be summarized as follows. By depending on this library via a Jenkins plugin, your users will get security updates as part of the regular plugin update process. When you bundle the library into the Fortify plugin, that benefit is lost.